src/libexec/zmcbpolicydinit

```diff
@@ -74,6 +74,35 @@ initCBPolicyd() {
   else
     echo "done."
   fi
+  
+  echo -n "Adding Domains..."
+    for dom in `zmprov gad`; do sqlite3 /opt/zimbra/data/cbpolicyd/db/cbpolicyd.sqlitedb "INSERT INTO policy_group_members (PolicyGroupID,Member) VALUES ('2','@$dom');"; done
+  if [ $? != 0 ]; then
+    echo "failed."
+    exit 1
+  else
+    echo "done."
+  fi
+
+ echo -n "Adding subnets from Mynetworks global..."
```

Comment:
nit: add one more space (should have two spaces preceding echo here for consistency)